src/components/Courses/component/CourseCard/CourseCard.jsx

```diff
@@ -7,31 +7,40 @@ import {
 import Button from '../../../../common/Button/Button';
 import { toHoursAndMinutes } from '../../../../helpers/timeConvert';
 
+import { useNavigate } from 'react-router-dom';
+
 import { useContext } from 'react';
 import { AuthorsContext } from '../../../../helpers/context/authors.context';
 
 const getStringWithAuthorsNames = (allAuthors, authorsIds, maxStringLength) => {
-	const namesArr = [];
-	authorsIds.forEach((authorId) => {
```

Comment:
This function feels like a helper. Move it please to keep this file related only to the component itself

Reply:
Oh, I missed it.
Yes, there are two, and they are very similar. I put it in the helpers